src/SimpleCalculator/Addition.cs

```diff
@@ -2,14 +2,24 @@ namespace SimpleCalculator;
 
 public class Addition
 {
-    UserInputs inputs = new UserInputs();
-    public int Add()
+    private double _firstNumber;
+    private double _secondNumber;
```

Comment:
looks like these private fields are not used. I'd suggest using private fields like this
```c#
public double FirstNumber 
{ 
 get { return _firstNumber; } 
}
```
or
```c#
public double FirstNumber { get; private set; }
```
to follow up the Encapsulation principle